plugins/lang-javascript/src/main/java/org/elasticsearch/script/javascript/JavaScriptScriptEngineService.java

```diff
@@ -68,7 +68,9 @@
  */
 public class JavaScriptScriptEngineService extends AbstractComponent implements ScriptEngineService {
 
-    public static final List<String> TYPES = Collections.unmodifiableList(Arrays.asList("js", "javascript"));
+    public static final String NAME = "javascript";
```

Comment:
It is and he declares it as the extension right below, right? This is the actual name of the language.

If you made it so that users could use any of the languages's extensions with the `lang` parameter then I think the `js` -> `javascript` change isn't breaking?